src/Foundatio/Queues/QueueEntry.cs

```diff
@@ -30,9 +30,9 @@ public QueueEntry(string id, string correlationId, T value, IQueue<T> queue, Dat
     public Type EntryType => Value.GetType();
     public object GetValue() => Value;
     public T Value { get; set; }
-    public DateTime EnqueuedTimeUtc { get; set; }
-    public DateTime RenewedTimeUtc { get; set; }
-    public DateTime DequeuedTimeUtc { get; set; }
+    public DateTimeOffset EnqueuedTimeUtc { get; set; }
+    public DateTimeOffset RenewedTimeUtc { get; set; }
+    public DateTimeOffset DequeuedTimeUtc { get; set; }
```

Comment:
I worry what this and the cache entry changes could break for existing cached / queued data.